RoomLocator/RoomLocator.Domain/ViewModels/ModcamInstallationsViewModel.cs

```diff
@@ -0,0 +1,12 @@
+namespace RoomLocator.Domain.ViewModels
+{
+    public class ModcamInstallationsViewModel
+    {
+
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string NameSpace { get; set; } // maybe not necessary to have it
```

Comment:
Kommentaren her er ligegyldig, bare fjern den. Hvis vi skal kigge på om vi skal have den fjernet, så opret et issue i stedet for. Ellers glemmer vi at kigge på det.